pkg/util/util.go

```diff
@@ -69,6 +69,14 @@ func logPanic(r interface{}) {
 	glog.Errorf("Recovered from panic: %#v (%v)\n%v", r, r, callers)
 }
 
+// Min will return the smaller of the 2 given integers.
```

Comment:
I have a medium-high preference to not have Min functions in our codebase; ```if a < b``` is not the kind of code duplication that's really harmful, just perhaps a bit ugly.

The email thread that convinced (a much younger version of) myself that min()/max() is just not a good fit for languages without generics can be found here: https://groups.google.com/forum/#!topic/golang-nuts/hT12Cno8k_E